src/components/Navbar/MenuItems.tsx

```diff
@@ -92,20 +101,21 @@ export function TopLeftNavItems({
       <NavMenuItem
         text={t`Projects`}
         onClick={onClickMenuItems}
-        route="/#/projects"
+        route="/projects"
       />
       <NavMenuItem
         text={t`FAQ`}
-        route={undefined}
+        route={'/#faq'}
```

Comment:
do we need this route and _also_ the onClick handler?

Reply:
Yeah I think we do - just had a look and it is being used for highlighting the menu bar on click on mobile:

https://user-images.githubusercontent.com/104132113/179432892-78290868-f281-4931-8ca2-9b7e42f14c39.mov